net-mgmt/pfSense-pkg-Telegraf/files/usr/local/pkg/telegraf.inc

```diff
@@ -51,12 +51,29 @@ function telegraf_resync_config() {
 		$telegraf_conf['interval'] = 10;
 	}
 
+	$shortname='';
+	if ($telegraf_conf['shortname'] == 'on') {
+		$shortname="\thostname = \"" . $config['system']['hostname'] . "\"";
+	}
+
+	$ca_pem_file = "/usr/local/etc/telegraf.ca";
+	$a_ca=$config['ca'];
+	if (!is_array($a_ca)) $a_ca=array();
```

Comment:
Please use [K&R, BSD KNF variant style](https://doc.pfsense.org/index.php/Developer_Style_Guide#Indent_style), even for simple statements, to keep them consistent and easy to read, and be careful to use matching and proper indentation.